api_tests/dry/rfc003_swap_reject.js

```diff
@@ -191,14 +227,16 @@ it("[Bob] Shows the swaps as Start in /swaps", async () => {
     let body = await bob.poll_comit_node_until(
         chai,
         "/swaps",
-        body => body._embedded.swaps.length == 2
+        body => body._embedded.swaps.length === 2
     );
 
     let swaps = body._embedded.swaps;
 
     for (let swap of swaps) {
         swap.protocol.should.equal("rfc003");
         swap.status.should.equal("IN_PROGRESS");
+        swap._links.accept.href.should.be.a("string");
```

Comment:
:x: I think we should just test that the properties "accept" and "decline" exist. No need to test if rustic-hal serializes correctly.